crates/router/src/connector/bambora.rs

```diff
@@ -0,0 +1,589 @@
+mod transformers;
+
+use std::fmt::Debug;
+
+use error_stack::{IntoReport, ResultExt};
+use transformers as bambora;
+
+use crate::{
+    configs::settings,
+    consts,
+    core::{
+        errors::{self, CustomResult},
+        payments,
+    },
+    headers,
+    services::{self, ConnectorIntegration},
+    types::{
+        self,
+        api::{self, ConnectorCommon, ConnectorCommonExt},
+        ErrorResponse, Response,
+    },
+    utils::{self, BytesExt},
+};
+
+#[derive(Debug, Clone)]
+pub struct Bambora;
+
+impl<Flow, Request, Response> ConnectorCommonExt<Flow, Request, Response> for Bambora
+where
+    Self: ConnectorIntegration<Flow, Request, Response>,
+{
+    fn build_headers(
+        &self,
+        _req: &types::RouterData<Flow, Request, Response>,
+        _connectors: &settings::Connectors,
+    ) -> CustomResult<Vec<(String, String)>, errors::ConnectorError> {
+        let mut headers = vec![(
+            headers::CONTENT_TYPE.to_string(),
+            self.get_content_type().to_string(),
+        )];
+        let mut api_key = self.get_auth_header(&_req.connector_auth_type)?;
+        headers.append(&mut api_key);
+        Ok(headers)
+    }
+}
+
+impl ConnectorCommon for Bambora {
+    fn id(&self) -> &'static str {
+        "bambora"
+    }
+
+    fn common_get_content_type(&self) -> &'static str {
+        "application/json"
```

Comment:
If you want to avoid spelling mistakes, `mime::APPLICATION_JSON` this can be used.